.github/workflows/make-rpm.yml

```diff
@@ -24,3 +24,12 @@ jobs:
           go-version: ${{ matrix.go }}
       - name: Build rpm
         run: CONTAINER_RUNTIME=docker make test-rpmbuild
+      - name: Create linux binary
+        run: make out/linux-amd64/crc
```

Comment:
There is already a binary built as part of `make test-rpmbuild`, you could copy it from the container image at the same time as the rpms
https://github.com/crc-org/crc/pull/3448/commits/8fdbf0b1f6790d82d66aec33886220ba4fa7b393#r1053324914